.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
files from .idea should not be pushed. plese, fix in the next pr

Reply:
roger that